react/Modal.tsx

```diff
@@ -64,7 +64,14 @@ const Modal: React.FC<Props> = props => {
     }
   }
 
-  const containerClasses = classnames(styles.containerCenter, handles.container, 'bg-base relative flex flex-column')
+  const containerClasses = classnames(
+    styles.containerCenter,
+    handles.container,
+    'bg-base relative flex flex-column',
+    {
+      [`${styles.fullScreenModal} w-100 mw-100 h-100 br0`]: fullScreen,
```

Comment:
Sounds a bit complicated imo.
```suggestion
      [styles.fullScreenModal]: fullScreen,
```